pep-0609.rst

```diff
@@ -0,0 +1,259 @@
+PEP: 609
+Title: PyPA Governance
+Version: $Revision$
+Last-Modified: $Date$
+Author: Dustin Ingram <di@python.org>,
+        Pradyun Gedam <pradyunsg@gmail.com>
+Sponsor: Paul Ganssle <paul@ganssle.io>
+BDFL-Delegate: TBD
+Discussions-To: https://discuss.python.org/t/closing-the-loop-on-pypa-governance-bdfrn/
```

Comment:
Should we create a new thread under the "PEPs" category for the PEP-specific discussion?

The upside is that it's a "clean slate" to comment specifically on the PEP. The downside is that the old discussion is not available in the main thread and some of it may get rehashed.

Reply:
Done. I'll actually create this thread once it's merged.